web_notify/static/src/js/services/web_notification_service.js

```diff
@@ -0,0 +1,54 @@
+/** @odoo-module **/
+
+import {browser} from "@web/core/browser/browser";
+import {ConnectionLostError} from "@web/core/network/rpc_service";
+import {registry} from "@web/core/registry";
+import ajax from "web.ajax";
+import session from "web.session";
+
+const webNotificationService = {
+    dependencies: ["action", "notification", "rpc"],
+
+    start(env, {action, notification, rpc}) {
+        ajax.loadJS("https://unpkg.com/sweetalert/dist/sweetalert.min.js");
+        env.bus.on("WEB_CLIENT_READY", null, async () => {
+            const legacyEnv = owl.Component.env;
+            legacyEnv.services.bus_service.onNotification(this, (notifications) => {
+                for (const {payload, type} of notifications) {
+                    if (type === "web_notify_channel") {
+                        displayWebNotification(payload);
+                    }
+                }
+            });
+            legacyEnv.services.bus_service.startPolling();
+        });
+
+        /**
+         * Displays the injected bus notification on user's screen
+         */
+        function displayWebNotification(notif) {
+            if (notif.beep) {
+                let audio = new Audio();
+                audio.src = session.url("/web_notify/static/audio/alert.mp3");
```

Comment:
Dont you have to introduce various beep for each type of type message ?
Otherwise, afaik, an alert.mp3 will be executed for success message. Dont you think ?

Reply:
hi,

that's a great idea thanks, I just updated the same.

Reply:
Hi what about to add also info and success beep ?
Otherwise a success action will send a warning sound. (If i undersrand correctly)

Also naming the files exactly as the level of the message could simplify the JS code.
Dont you think ?

Reply:
as per the availability of the audio file i just only added these changes if u have some audio sources similar to this plz share them with me I will make the necessary changes.

Reply:
u can find me on 👍 https://www.linkedin.com/in/iamaswanth/